src/participant/prompts/exportToPlayground.ts

```diff
@@ -0,0 +1,32 @@
+import * as vscode from 'vscode';
+
+export class EportToPlaygroundPrompt {
+  static getAssistantPrompt(): vscode.LanguageModelChatMessage {
+    const prompt = `You are a MongoDB expert.
+Your task is to help the user build MongoDB queries and aggregation pipelines that perform their task.
+You achieve this by converting user's code written in any proggramming language to the MongoDB Shell syntax.
+Take a user promt as an input string and translate it to the MongoDB Shell language.
+Keep your response concise.
+You should suggest queries that are performant and correct.
+Respond with markdown, suggest code in a Markdown code block that begins with \`\`\`javascript and ends with \`\`\`.
+You can imagine the schema, collection, and database name.
```

Comment:
I wonder if this may hurt the ai's generations. It seems like it invites more unpredictable outputs. Would it make sense to remove it or does it help certain situations?
```suggestion
```